ansible/roles/test/vars/testcases.yml

```diff
@@ -27,6 +27,10 @@ testcases:
           ptf_host:
           testbed_type:
 
+    bgp_neighbor:
```

Comment:
bgp_neighbor_flap
I think we can rename the test, so that it reflects flap part. Thx